TerminalDocs/customize-settings/profile-general.md

```diff
@@ -95,7 +93,12 @@ This sets the icon that displays within the tab, dropdown menu, jumplist, and ta
 
 **Accepts:** File location as a string, or an emoji
 
-<br />
+The icon image file will need to be placed in a location that the Terminal app can read. Place icon images in the Windows Terminal AppData folder, located at: <br>
+
+`%LOCALAPPDATA%\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\RoamingState`
+<br>
+<br>
+As an example, if you place the icon image `ubuntu.ico` in this folder, the property will be `"icon": "ms-appdata:///roaming/ubuntu.ico"`.
```

Comment:
This is not true, icon image files can be anywhere on the machine.

Reply:
Should we show a couple of different examples that illustrate this to reduce confusion?

Reply:
I just changed this from a descriptor to an example - how does this look? https://github.com/MicrosoftDocs/terminal/commit/65c6d39fa227b38f63415fa9d818f0622aabf79b